### DISCUSSION TITLE
Markdown rendering fails at image insertion and mathematical input

### DISCUSSION THREAD

**Discussion by Yonggie** [academicpages/academicpages.github.io#2677] [Q&A]
Math failure:
![图片](https://github.com/user-attachments/assets/f3019c22-bacc-4642-9fd0-72740e67b842)
![图片](https://github.com/user-attachments/assets/6ad98469-9dab-4d01-ad96-f58d94ae46a1)

Image failure:
![图片](https://github.com/user-attachments/assets/e574a49b-1d92-495e-a5bc-a2f1b6ec7913)
![图片](https://github.com/user-attachments/assets/88f15a0a-e16c-4078-a47a-e4951c8465ae)

**Comment by Yonggie** [comment]
For question 2, I have figured that I have to use ``/files/image_name.png`` to let the project find the right img_path.

Any help on question 1?

**Reply by rjzupkoii** [reply]
Per the [Guide](https://academicpages.github.io/markdown/) inline MathJax notation is `$$`, so it should be `$$k$$`.
